scripts/gen_uber_class.ts

```diff
@@ -0,0 +1,293 @@
+import * as ts from 'typescript';
+const fs = require('fs');
+// TODO: maintainer - if a new client is added, add it to this list
+
+// Intentionally not surfacing projectServiceClient - it has methods that do not follow these patterns
+const clients = [
+  'DatasetServiceClient',
+  'TableServiceClient',
+  'JobServiceClient',
+  'ModelServiceClient',
+  'RoutineServiceClient',
+  'RowAccessPolicyServiceClient',
+];
+const files = [
+  '../src/v2/dataset_service_client.ts',
+  '../src/v2/table_service_client.ts',
+  '../src/v2/job_service_client.ts',
+  '../src/v2/model_service_client.ts',
+  '../src/v2/routine_service_client.ts',
+  '../src/v2/row_access_policy_service_client.ts',
+];
+const output = `
+// Copyright 2025 Google LLC
```

Comment:
Check on if this will need to be updated yearly

Reply:
per our internal policies we do not update this yearly - it should remain the year it was first published